apps/src/bin/namada-node/cli.rs

```diff
@@ -57,6 +58,24 @@ pub fn main() -> Result<()> {
                     );
                 }
             }
+            cmds::Config::UpdateLocalConfig(cmds::LocalConfig(args)) => {
+                // Validate the new config
+                let updated_config = std::fs::read(args.config_path).unwrap();
+                let _validator_local_config: ValidatorLocalConfig =
+                    toml::from_slice(&updated_config).unwrap();
+
+                // Update validator configuration file with the new accepted
```

Comment:
presumably other updates would also be applied, if present in the file read?

Reply:
this is fine, but we should make sure the comments and cli usage info match what the code actually does